packages/abi-coder/src/coders/byte.ts

```diff
@@ -0,0 +1,62 @@
+import { arrayify, concat, type BytesLike } from '@ethersproject/bytes';
```

Comment:
```suggestion
import { arrayify, concat } from '@ethersproject/bytes';
```

The linting is saying `BytesLike` is unused.

Reply:
I committed it to shorten the review cycle.